numpy/core/src/multiarray/conversion_utils.c

```diff
@@ -945,14 +951,20 @@ PyArray_IntpFromIndexSequence(PyObject *seq, npy_intp *vals, npy_intp maxvals)
 }
 
 /*NUMPY_API
- * PyArray_IntpFromSequence
+ * PyArray_IntpFromSequence`
  * Returns the number of integers converted or -1 if an error occurred.
  * vals must be large enough to hold maxvals
  */
 NPY_NO_EXPORT int
 PyArray_IntpFromSequence(PyObject *seq, npy_intp *vals, int maxvals)
 {
-    return PyArray_IntpFromIndexSequence(seq, vals, (npy_intp)maxvals);
+    npy_intp result = PyArray_IntpFromIndexSequence(
+        seq, vals, (npy_intp)maxvals);
```

Comment:
Hmm, this might look broken and aligned.

Reply:
Are you saying the indentation is wrong?

Reply:
No, just that it might look better if it started after `(` in the usual way. However, it looks like some of these files never got the full style cleanup done many years ago, so probably we can do all that in one swoop at a future date.